packages/runtime/test/consumption/openid4vc.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
This may hide real problems in this file. I consciously decided to go with `eslint-disable-next-line` in my last PR.